excludes/pt_BR.php

```diff
@@ -87,6 +87,7 @@
     'Tokelau',
     'Tonga',
     'total',
+    'Total'
```

Comment:
The word `Total:` must be followed by a colon.

https://github.com/Laravel-Lang/lang/blob/master/locales/pt_BR/pt_BR.json#L636

Reply:
```suggestion
    'Total:'
```

Reply:
```suggestion
    'Total:'
```